transcom/mymove
# INT B-19226 Change forms to indicate "required" fields instead of "optional" ones

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19226)

## Summary

This PR changes forms in MilMove to label required fields with light grey "Required" text, while getting rid of the existing "Optional" text that currently accompanies optional fields. Also made some changes to components like <DutyLocationInput> to make them behave similar to other components, mostly in that they will glow red with warning text if the user clicks out of them without entering data.

### How to test

1. Login as customer and go through all onboarding and create a shipment. Ensure that all fields that are required have "required" in grey text beside them, and alert the user in red if they click out without entering data.
2. Ensure that no fields are show the "Optional" tag, only "Required" fields should have light grey help text.

## Screenshots
![New form examplepng](https://github.com/user-attachments/assets/ffd95d66-b796-47c8-a34f-54e7c5113388)
![field warning example](https://github.com/user-attachments/assets/fab2a4fb-8f8b-4ba6-b458-76de10e7a7c4)
